.github/workflows/go.yml

```diff
@@ -56,3 +56,33 @@ jobs:
       run: |
         go test -timeout 60s -cover \
             $(go list ./config/... ./validate/...) --race
+  schema:
+    name: schema
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out repository
+      uses: actions/checkout@v3
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v3
+      with:
+        go-version: 1.20.x
+    - name: Install schematyper
+      run: |
+        # "go install github.com/idubinskiy/schematyper:latest" fails with
+        # current Go.  Use fix from fork.  We can't "go install" directly from
+        # the fork; it complains about mismatched package paths.
+        # https://github.com/idubinskiy/schematyper/pull/22
+        git clone https://github.com/bgilbert/schematyper
+        cd schematyper
+        git checkout gomod
```

Comment:
Is this step required? Why would `gomod` have any changes by this point?

Reply:
We want the branch of the fork with the fix in it, not the `main` branch.  But actually we can do that as part of `clone`.  Updated.